ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/provision/cucushift-installer-rehearse-vsphere-ipi-zones-provision-chain.yaml

```diff
@@ -6,6 +6,7 @@ chain:
   - ref: enable-qe-catalogsource
   - chain: cucushift-installer-check
   - ref: cucushift-installer-check-vsphere-cpms
+  - ref: cucushift-installer-check-vsphere-zone
```

Comment:
Can we also add this step to `cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb-provision` ?

Reply:
since we already changed install-config with failureDomain format in regular IPI install on 4.13+ via https://github.com/openshift/release/pull/50162/, I guess we can also add this step to job running on 4.13+? for example, `cucushift-installer-rehearse-vsphere-ipi-static-ip-provision-chain`, I see related jobs run on 4.14+.

Reply:
good catch. let me update and run related job. the job for single zone seems fine.